microsoft-edge/webview2/concepts/distribution.md

```diff
@@ -6,10 +6,20 @@ ms.author: msedgedevrel
 ms.topic: conceptual
 ms.prod: microsoft-edge
 ms.technology: webview
-ms.date: 3/29/2023
+ms.date: 08/21/2023
 ---
 # Distribute your app and the WebView2 Runtime
 
+<!-- todo: 
```

Comment:
@liminzhu - you can push a commit to incorp this "todo"

Reply:
Can you add a line at the end of #### Deploying the Evergreen WebView2 Runtime and before  #### Installing the Runtime as per-machine or per-user - 

```
For developers using [App Installer](https://learn.microsoft.com/windows/msix/app-installer/app-installer-root) to deploy your MSIX applications, you can specify WebView2 runtime as a dependency and have it installed with the application. More details in App Installer [documentation](https://learn.microsoft.com/uwp/schemas/appxpackage/uapmanifestschema/element-win32dependencies-externaldependency).
```

Reply:
*  If you're using App Installer to deploy MSIX applications, you can specify the WebView2 Runtime as a dependency, to have it be installed with the application, instead of using the Evergreen Bootstrapper, Evergreen Standalone Installer, or Fixed Version (Self-Contained) distribution.  For details about how to do this, see [win32dependencies:ExternalDependency (Windows 10, Windows 11)](/uwp/schemas/appxpackage/uapmanifestschema/element-win32dependencies-externaldependency) in the App package manifest docs.  See also [Install Windows 10 apps with App Installer](/windows/msix/app-installer/app-installer-root).